Tomas Heran <tomas.heran@oracle.com>
This patch will not be reported upstream.

diff -r 2b3bbde72bcd -r 92407d8c9b87 doc/libmicrohttpd.3
--- a/doc/libmicrohttpd.3	Mon Jan 26 16:56:08 2015 +0100
+++ b/doc/libmicrohttpd.3	Thu Feb 05 13:03:58 2015 +0100
@@ -33,3 +33,6 @@
 
 .SH "AVAILABILITY"
 You can obtain the latest version from http://www.gnu.org/software/libmicrohttpd/.
+
+.SH "CAVEATS"
+This library, as delivered by Oracle Solaris in binary form, has been compiled without HTTPS support, i.e. \fB--disable-https\fP option was used when configuring the library for compilation.
